praw/models/reddit/subreddit.py

```diff
@@ -1545,6 +1545,50 @@ def __init__(self, subreddit):
         """Construct an instance of the Modmail object."""
         self.subreddit = subreddit
 
+    def conversations(self, after=None, limit=25, other_subreddits=None,
+                      sort=None, state=None):
+        """Return a list of conversations.
+
+        Example:
+
+        .. code:: python
+
+            conversations = reddit.subreddit('all').conversations(state='mod')
+
+        :param after: A :class:`.ModmailConversation` instance. When provided,
+            the listing begins after this conversation (default: None).
+        :param limit: The maximum number of conversations to fetch
+            (default: 25).
+        :param other_subreddits: A list of `.Subreddit` instances for which to
+            fetch conversations (default: None).
+        :param sort: Can be one of: recent, mod, user, unread
+            (default: recent).
+        :param state: Can be one of: new, inprogress, mod, notifications,
+            archived, highlighted, all (default: all).
+
+        """
+        params = {}
+        if self.subreddit.display_name != "all":
+            if other_subreddits is None:
+                other_subreddits = []
+            subreddits = [self.subreddit] + other_subreddits
+            params['entity'] = ",".join(subreddit.display_name
+                                        for subreddit in subreddits)
+
+        if after:
+            params['after'] = after.id
```

Comment:
I believe elsewhere after is used, it must be the id, and not an object. For consistency, can we require that the ID is passed in? If `str(conversation)` outputs the ID, then it would just-so-happen that the conversation can be passed in.